src/components/shared/DomainFundSelectorSection/DomainFundSelectorSection.tsx

```diff
@@ -38,38 +38,23 @@ const DomainFundSelectorSection = ({
   transferBetweenDomains,
   disabled,
 }: Props) => {
-  const {
-    getValues,
-    setValue,
-    formState: { errors },
-    clearErrors,
-  } = useFormContext();
+  const { getValues, setValue, trigger } = useFormContext();
   const values = getValues();
   const handleFromDomainChange = (fromDomainValue) => {
     if (values.motionDomainId !== fromDomainValue) {
       setValue('motionDomainId', fromDomainValue);
     }
+    trigger();
```

Comment:
Changing from domain should trigger the validation of both toDomainId (to check if it's not the same domain) and amount (a previously valid amount might not be valid for the new domain anymore)